src/main/java/ua/cv/tim/service/impl/AllianceServiceImpl.java

```diff
@@ -109,9 +110,9 @@ public Alliance getById(String  uuid){
     public Alliance getByName(String name) {
         return allianceDao.getByName(name, null);
     }
-
+    @Override
     public boolean isUniqueAlliance(String name, String uuid){
-
+        logger.info("Alliance isUniqueAlliance");
```

Comment:
redundant logging because logger print class and methods names automaticaly.

in this case better to logging some value, that you want to check